aws/sdk/examples/medialive-helloworld/src/main.rs

```diff
@@ -0,0 +1,16 @@
+#[tokio::main]
+async fn main() -> Result<(), medialive::Error> {
+    let client = medialive::Client::from_env();
+    let input_list = client.list_inputs().send().await?;
+
+    if let Some(inputs) = input_list.inputs {
```

Comment:
the pattern we use a lot for this is `input_list.unwrap_or_default()`

Reply:
Like this ?

`input_list.inputs.unwrap_or_default().iter().for_each(f)...`

Reply:
added github suggestion